waspc/ChangeLog.md

```diff
@@ -1,11 +1,15 @@
 # Changelog
 
-## Next
+## [WIP] 0.16.0
```

Comment:
What was wrong with Next? 😢 I kinda liked it, you don't have to think about what the next version will be

Reply:
Next is cool! But in this case I knew what next version is going to be, so I thought why not write it down? But yeah I could have left Next you are right, wouldn't be an issue.